packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,215 @@
+                  <Stack horizontal className={phoneInfoStep}>
+                    <Stack.Item className={phoneInfoIcon(theme)}>
+                      <Stack verticalAlign="center" horizontalAlign="center">
+                        <Icon
+                          iconName="PhoneNumberButton"
+                          style={{ color: theme.palette.themePrimary, padding: '8px' }}
+                        />
+                      </Stack>
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalDialIn}</Text>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Stack.Item className={phoneInfoStep}>
+                  {conferencePhoneInfoList.map((phoneNumber, index) => (
+                    <Stack.Item key={index}>
+                      <Text className={phoneInfoTextStyle}>{formatPhoneNumberInfo(phoneNumber, strings)}</Text>
+                    </Stack.Item>
+                  ))}
+                </Stack.Item>
+              </Stack>
+              <Stack
+                horizontal
+                horizontalAlign="space-between"
+                verticalAlign="center"
+                className={phoneInfoInctructionLine}
+              >
+                {}
+                <Stack.Item>
+                  <Stack horizontal>
+                    <Stack.Item className={phoneInfoIcon(theme)}>
+                      <Icon
+                        iconName="DtmfDialpadButton"
+                        style={{ color: theme.palette.themePrimary, padding: '8px' }}
+                      />
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalMeetingId}</Text>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Text className={phoneInfoTextStyle}>{conferencePhoneInfoList[0].conferenceId}#</Text>
+              </Stack>
+              <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
+                <Stack horizontal>
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Icon iconName="PhoneInfoWait" style={{ color: theme.palette.themePrimary, padding: '8px' }} />
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalWait}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack>
+            </Stack>
+          )}
+        </Modal>
+      }
+    </>
+  );
+};
+
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  const cleaned = ('' + phoneNumber).replace(/\D/g, '');
+  const match = cleaned.match(/^(\d{1,2})?(\d{3})(\d{3})(\d{4})$/);
```

Comment:
Will all 'join by phone' numbers conform to `+# (###) ### - ###` - this feels very specific to north america

Reply:
I've looked on other patterns on this page https://dialin.teams.microsoft.com/8551f4c1-bea3-441a-8738-69aa517a91c5?id=5969050
Current logic is to format 11 and 12 digits numbers with +# (###) ### - ###, could be more common without brackets
But if number is bigger or smaller it will not be formatted.

Unfortunately tsCalling is providing plain string.

Reply:
Super handy link! Yeah looks like most common without brackets. Are there any libraries out there that would do the formatting for us? i.e. be smarter and check the start for a known country code then format the string appropriately?

Reply:
(perhaps this? https://github.com/catamphetamine/libphonenumber-js)